modules/4337/certora/conf/Safe4337Module.conf

```diff
@@ -18,6 +18,6 @@
     "verify": "Safe4337Module:certora/specs/Safe4337Module.spec",
     "packages": [
         "@account-abstraction=../../node_modules/.pnpm/@account-abstraction+contracts@0.7.0/node_modules/@account-abstraction",
-        "@safe-global=../../node_modules/.pnpm/@safe-global+safe-contracts@1.4.1-build.0_ethers@6.13.1_bufferutil@4.0.8_utf-8-validate@6.0.4_/node_modules/@safe-global"
+        "@safe-global=../../node_modules/.pnpm/@safe-global+safe-contracts@1.4.1-build.0_ethers@6.13.1_bufferutil@4.0.8_utf-8-validate@5.0.10_/node_modules/@safe-global"
```

Comment:
I think this is wrong... the dependencies changed a bit recently. Did you `pnpm install`?